Libraries/LibCore/CHttpJob.cpp

```diff
@@ -4,8 +4,37 @@
 #include <stdio.h>
 #include <unistd.h>
 
+#include <CGzip.h>
+
 #define CHTTPJOB_DEBUG
 
+static AK::ByteBuffer handle_content_encoding(const AK::ByteBuffer& buf, const AK::String& content_encoding)
```

Comment:
No need to qualify things with `AK::`, all `AK` headers pull things into the global namespace.
Also, please write out full words like "buffer"